### DISCUSSION TITLE
fa/manager/configuration-and-advanced-settings/How-to-set-up-Telegram-proxy-on-Hiddify/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4696] [Content Help کمک به اصلاح محتوا]
# fa/manager/configuration-and-advanced-settings/How-to-set-up-Telegram-proxy-on-Hiddify/

The proxy manager platform.

https://hiddify.com/fa/manager/configuration-and-advanced-settings/How-to-set-up-Telegram-proxy-on-Hiddify/?h=%D9%BE%D8%B1%D9%88%DA%A9%D8%B3%DB%8C+%D8%AA

**Comment by hiwasam** [comment]
درود
مشکلی که هست حچم مصرفی پروکسی تلگرام در ترافیک کاربر حساب نمیشه
ایا راه حلی داره؟
